### PR TITLE
Add logic to stop a consumer after a number of seconds

### PR DESCRIPTION
**Pull request by mihaileu**
add logic to stop a consumer after a specific time

### CODE REVIEW

**Comment by mihaileu** [comment]
@mateusjunges can you take a look ?!

File: src/Consumers/Consumer.php [hunk 1]
```diff
@@ -350,6 +352,17 @@ private function maxMessagesLimitReached(): bool
         return $this->messageCounter->maxMessagesLimitReached();
     }
 
+    public function configureStopTimer(): Timer
+    {
+        $stopTimer = new Timer();
+        if ($this->config->getMaxTime() == 0) {
+            $stopTimer = new InfiniteTimer();
+        }
+        $stopTimer->start($this->config->getMaxTime() * 1000);
```
Comment (mateusjunges) [inline src/Consumers/Consumer.php:361]:
```suggestion
        }

        $stopTimer->start($this->config->getMaxTime() * 1000);
```

File: src/Config/Config.php [hunk 2]
```diff
@@ -113,7 +114,10 @@ public function getMaxMessages(): int
     {
         return $this->maxMessages;
     }
-
+    public function getMaxTime(): int
+    {
+        return $this->maxTime;
+    }
```
Comment (mateusjunges) [inline src/Config/Config.php:120]:
```suggestion
    }

```

**Comment by mihaileu** [comment]
> @mihaileu How do I set the max time when creating a consumer using the `Kafka` facade?

added missing facade method

**Review by mateusjunges — APPROVED** [review]

**Pull request merged**